backend/src/zango/apps/shared/tenancy/templatetags/zango_filters.py

```diff
@@ -36,3 +39,16 @@ def humanize_timedelta(timedelta_obj):
     if seconds > 0:
         time_total += " {} seconds".format(int(seconds))
     return time_total
+
+
+@register.filter()
+def update_latest(build_path):
+    buildir = os.path.dirname(build_path)
+    filep = os.path.basename(build_path)
+    for dir in settings.STATICFILES_DIRS:
+        for dirpath, dirnames, filenames in os.walk(dir):
+            for filename in filenames:
+                if re.match(filep, filename):
+                    print(os.path.join(dirpath, filename))
```

Comment:
we can remove print statement

we can add some docstring in this function mentioning how it expects and handles the input path